src/seedu/addressbook/common/Utils.java

```diff
@@ -34,4 +36,23 @@ public static boolean elementsAreUnique(Collection<?> items) {
         }
         return true;
     }
+
+    /**
+     * Returns a concatenated version of printableStrings
+     * @param printables objects to print
+     */
+    public static String getPrintableString(Printable... printables) {
+        StringBuilder toPrint = new StringBuilder();
+        final String detailIsPrivate = "(private) ";
+
+        for (Printable p : printables) {
+            if (p.isPrivate()) {
+                toPrint.append(detailIsPrivate);
```

Comment:
:bulb: *Java tip*: Java 8 has a `StringJoiner` class that can join multiple strings using a delimiter e.g. `aaa` `bbb` `ccc` --> `aaa, bbb, ccc` https://www.mkyong.com/java8/java-8-stringjoiner-example/